playroom/snippets.tsx

```diff
@@ -1076,6 +1076,70 @@ const navigationBarSnippets = [
     },
 ];
 
+const carouselSnippets = [
+    {
+        group: 'Carousel',
+        name: 'Carousel',
+        code: `
+<Carousel
+  withBullets
+  items={Array.from({ length: 6 }, (_, idx) => (
+    <MediaCard
+      headline={<Tag type="promo">Headline</Tag>}
+      title={'Card ' + idx}
+      description="Description"
+      media={<Image src="https://i.imgur.com/flZfkiX.png" aspectRatio="16:9" />}
+      button={
+        <ButtonPrimary small onPress={() => {}}>
+          Action
+        </ButtonPrimary>
+      }
+      buttonLink={<ButtonLink onPress={() => {}}>Link</ButtonLink>}
+    />
+  ))}
+/>`,
+    },
+    {
+        group: 'Carousel',
+        name: 'FullWidthCarousel',
+        code: `
+<FullWidthCarousel
+  withBullets
+  items={[
+    <Image src="https://i.imgur.com/HRvhZ6F.jpeg" aspectRatio="16:9" />,
+    <Image src="https://i.imgur.com/flZfkiX.png" aspectRatio="16:9" />,
+  ]}
+/>`,
+    },
+    {
+        group: 'Carousel',
+        name: 'CenteredCarousel',
+        code: `
+<CenteredCarousel
+  withBullets
```

Comment:
props help legend is not appearing in playroom 🤔

Reply:
hmmm... weird. I'll take a look

Reply:
it seems to work for `Slideshow` and `CenteredCarousel`, but not for `Carousel`. I don't know why